entity-framework/core/get-started/configuring-dbcontext.md

```diff
@@ -158,6 +152,12 @@ using (var context = serviceProvider.GetService<BloggingContext>())
 var options = serviceProvider.GetService<DbContextOptions<BloggingContext>>();
 ```
 
+## Design-time DbContext configuration
```

Comment:
I wonder if we should just remove this entirely and link to the design-time dbcontext creation page at the bottom.

Reply:
Since this page centralizes the various ways contexts can be configured, it makes sense to me to have this section (which is small and links to the full page (though I did move it to the bottom here since it really doesn't seem like the first thing users should see). But am also OK to eliminate and just link...